test/onnx/test_utility_funs.py

```diff
@@ -186,7 +186,6 @@ def forward(self, x):
 
         for node in graph.nodes():
             self.assertNotEqual(node.kind(), "onnx::ReduceL2")
-        self.assertEqual(len(list(graph.nodes())), 2)
```

Comment:
@BowenBao thank you, I applied your suggestions to the failing expecttests. I also needed to remove this assert equal give that, since now `norm` is composite, it so happens that the length of the decomposition changes between ONNX versions (it changes from version 12 to 13). I hope that this is fine.